gammapy/modeling/models/tests/test_cube.py

```diff
@@ -532,6 +532,18 @@ def test_compute_npred(diffuse_evaluator):
         assert_allclose(out.data.sum(), 1.106403e12, rtol=5e-3)
         assert_allclose(out.data[0, 0, 0], 8.778828e09, rtol=5e-3)
 
+    @staticmethod
+    def test_apply_edisp_false(diffuse_evaluator):
+        # TODO: needs_update=True if apply_irf change ?
+        diffuse_evaluator.model.apply_irf["edisp"] = False
+        diffuse_evaluator.use_cache = False
+        del diffuse_evaluator.methods_sequence
```

Comment:
Why do you change these settings here?

Reply:
I would rather recreate a `MapEvaluator` as modifying a fixture might cause issues depending on the order the tests are run.

Reply:
Could this rather be tested in test_evaluator.py directly? It does not necessarily need a template model, right?

Reply:
ok I created a separate fixture, but I would prefer to keep the test here to avoid core duplication recreating psf/edisp/model.